pinot-spi/src/main/java/org/apache/pinot/spi/utils/CommonConstants.java

```diff
@@ -2091,6 +2090,10 @@ public static class FieldSpecConfigs {
         "pinot.field.spec.default.json.max.length";
   }
 
+  public static class IngestionConfigs {
+    public static final int DEFAULT_INGESTION_EXCEPTION_LOG_RATE_LIMIT_PER_MIN = 5;
```

Comment:
@Jackie-Jiang 
note: This is a logging behavior change for transformer exception throws. meaning that first 5 logs will be `WARN` or `ERROR` per min